### DISCUSSION TITLE
PDF document thumbnails & preview

### DISCUSSION THREAD

**Discussion by clstaudt** [flet-dev/flet#822] [Q&A] [answered]
Can I use flet to:

- display thumbnails of a set of PDF documents?
- display a preview of the selected PDF document?

<img width="1124" alt="Screenshot 2023-01-07 at 13 08 14" src="https://user-images.githubusercontent.com/875194/211149916-4e77cbe1-85f5-4fb4-85fd-d493376cdd09.png">

**Comment by FeodorFitsner** [comment, marked as answer]
I think you need to generate thumbnails in Python using some library and then display that image in Flet.
